content/md/imports-exports/monitor-jobs_fr.md

```diff
@@ -0,0 +1,68 @@
+---
+id: monitor-jobs
+themes: import-export-data, administration
+title: Suivez les exécutions des **traitements**
+popular: false
+ee-only: false
+related: imports, exports, product-export-builder, discover-the-dashboard, what-is-a-rule, what-is-a-project, product-mass-actions
+---
+
+# Qu'est-ce qu'un traitement ?
+
+Un traitement peut être :
+- Un [import](imports.html),
+- Un [export](exports.html),
+- Une [édition en masse](product-mass-actions.html),
+- Une exécution de [règle](what-is-a-rule.html) _(EE uniquement)_,
+- Un calcul de [projet](what-is-a-project.html)  _(EE uniquement)_.
+
+Vous avez plusieurs façons de suivre le statut d'un traitement.
+
+# Comment surveiller ?
+Tout d'abord, pour les actions en tache de fond comme une modification de masse ou une exécution de règle, vous êtes averti par un message flash lorsque le travail commence.
+![image](Settings_Calculation1_fr.png)
+
+Ensuite, lorsque le travail est terminé, une notification apparaît en haut de votre écran.
+![image](Settings_Calculation2_fr.png)
+
+Il existe une page de suivi des processus qui vous donnera la liste complète de tous les travaux que vous avez lancés. Pour y accéder, allez dans `Activité` et cliquez sur l'entrée du menu `Traqueur de processus`.
+![image](Settings_Calculation3_fr.png)
+
+Le widget `Dernières opérations` sur le tableau de bord Akeneo vous indiquera également le statut de vos travaux.
+![image](Dashboard_Calculation4_fr.png)
```

Comment:
Shouldn't the status be translated in French too?

Reply:
New screenshot sent